app/tests/importers/test_wikifactory_importer.py

```diff
@@ -0,0 +1,221 @@
+import os
+from typing import Any, Dict, Generator, List
+
+import gql
+import py
+import pytest
+import requests
+from gql import Client
+from requests.models import Response
+from sqlalchemy.orm import Session
+
+from app import crud
+from app.importers.wikifactory import WikifactoryImporter, space_slug_from_url
+from app.models.job import JobStatus
+from app.models.job_log import JobLog
+from app.models.manifest import Manifest
+from app.schemas import JobCreate
+from app.tests.utils import utils
+
+
+@pytest.mark.parametrize(
+    "project_url, space, slug",
+    [
+        (
+            "http://wikifactory.com/@botler/test-project",
+            "@botler",
+            "test-project",
+        ),
+        ("http://wikifactory.com/+wikifactory/试验", "+wikifactory", "试验"),
+    ],
+)
+def test_space_slug_from_url(project_url: str, space: str, slug: str) -> None:
+    result = space_slug_from_url(project_url)
+    assert result.get("space") == space
```

Comment:
It's pointless to use `get()` when you expect something to be there
```suggestion
    assert result["space"] == space
```